nixos/modules/services/continuous-integration/github-runner.nix

```diff
@@ -120,13 +120,7 @@ in
         RUNNER_ROOT = runtimeDir;
       };
 
-      path = (with pkgs; [
-        bash
-        coreutils
-        git
-        gnutar
-        gzip
-      ]) ++ [
+      path = [
```

Comment:
It's not clear to me why those have been removed.

Reply:
While those packages are not strictly necessary to run the GitHub Actions Runner, they are the bare minimum for the [checkout action](https://github.com/actions/checkout).

Reply:
This would mean that basically everything is broken, right?

Reply:
Not necessarily. You might be using the runner without the checkout action—I just don't think this is a common use case. If we leave those packages out, everybody else has to figure out the packages required to use the checkout action themselves. I'd definitively keep them.